cmake/configure.cmake

```diff
@@ -82,11 +82,12 @@ if(WITH_GOLANG)
   file(MAKE_DIRECTORY "${PADDLE_IN_GOPATH}")
   set(PADDLE_GO_PATH "${CMAKE_SOURCE_DIR}/go")
 
-  add_custom_target(go_path)
-  add_custom_command(TARGET go_path
+  message(STATUS "link ${CMAKE_SOURCE_DIR} to ${PADDLE_IN_GOPATH}...")
+
+  execute_process(
```

Comment:
I think the `execute_process` will only run when cmake is executed (when running `cmake ..`, or when cmake file changed). But it will not run every time make is called, or when `glide.lock` is changed.
This is an interesting problem, I gave it a try as well, can you take a look? https://github.com/PaddlePaddle/Paddle/pull/2884/